src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea, FigureInfo {
+    private String color;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }

```

Comment:
you should initialise color using constructor of Figure, that you should call inside of every its heirs